packages/framework/src/Form/Admin/SalesRepresentative/SalesRepresentativeFormType.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Form\Admin\SalesRepresentative;
+
+use Shopsys\FrameworkBundle\Form\Constraints\Email;
+use Shopsys\FrameworkBundle\Form\DisplayOnlyType;
+use Shopsys\FrameworkBundle\Form\GroupType;
+use Shopsys\FrameworkBundle\Model\SalesRepresentative\SalesRepresentative;
+use Shopsys\FrameworkBundle\Model\SalesRepresentative\SalesRepresentativeData;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\EmailType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints;
+
+class SalesRepresentativeFormType extends AbstractType
+{
+    private ?SalesRepresentative $salesRepresentative = null;
+
+    public function __construct()
+    {
+    }
+
+    /**
+     * @param \Symfony\Component\Form\FormBuilderInterface $builder
+     * @param array $options
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
```

Comment:
![image](https://github.com/user-attachments/assets/5d4ce79e-3324-4596-871c-6d2e95e18611)

based on the user story, everything should be nullable, has this changed? If so, please update the user story, if not, please modify the code accordingly (entity, form constraints, etc.)

Reply:
done